src/app/controllers/CartController.ts

```diff
@@ -492,4 +494,65 @@ export class CartController {
           });
     });
   }
+
+  /**
+   * Set host to pay for the entire order
+   * @param {string} cartId
+   * @returns {Promise<Cart>}
```

Comment:
Does return the cart or just cartId?

Reply:
I can change it so it returns string, the GraphQL returns the entire cart though,

would removing the 
`{
        _id
    }` return the entire cart?

Reply:
I'm not sure I feel like this just forces the ID to be returned only. GraphQL might return the whole thing or ALLOW you to return the whole thing but you are only quering the ID so you are only asking for hte Cart ID. Does that make sense?

Reply:
Yep I'm wondering if there's a way to return the entire cart object from ts-sdk, right now it's just id like you said. I can change it to expect a string to return but I remember Andrew saying it would be better to return the entire cart